cloudformation/template.yaml

```diff
@@ -3,10 +3,39 @@ AWSTemplateFormatVersion: '2010-09-09'
 Transform: 'AWS::Serverless-2016-10-31'
 
 Description: My Dev Portal Stack
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      -
+        Label:
+          default: "Dev Portal Content Configuration"
+        Parameters:
+          - ArtifactsS3BucketName
+          - DevPortalSiteS3BucketName
+          - StaticAssetRebuildToken
+          - StaticAssetRebuildMode
+      -
+        Label:
+          default: "Dev Portal Customer Configuration"
+        Parameters:
+          - CognitoIdentityPoolName
+          - DevPortalCustomersTableName
+      -
+        Label:
+          default: "Subscription Notification Configuration"
+        Parameters:
+          - MarketplaceSubscriptionTopicProductCode
+      -
+        Label:
+          default: "Custom Domain Configuration"
+        Parameters:
+          - CustomDomainName
+          - CustomDomainNameAcmCertArn
+          - UseRoute53Nameservers
 Parameters:
   ArtifactsS3BucketName:
     Type: String
-    Description: The S3 bucket in which the Lambda function code is stored. Bucket
```

Comment:
Confusingly enough, this S3 bucket is actually for swagger/OAS3 documents. There's a work item to rename all references to the buckets, because right now they have different names in different places, and the names aren't particularly descriptive. The lambda function code bucket is the "first" bucket that you pass to the `sam package` command, and isn't actually in this template (although it is in the `codeUri` properties in the packaged/output template).

Reply:
Whoops, read the diff wrong. Change looks correct; thanks for the doc fix! :)